src/wallet/coinselection.cpp

```diff
@@ -49,28 +49,25 @@ struct {
  * @param const std::vector<CInputCoin>& utxo_pool The set of UTXOs that we are choosing from.
  *        These UTXOs will be sorted in descending order by effective value and the CInputCoins'
  *        values are their effective values.
- * @param const CAmount& target_value This is the value that we want to select. It is the lower
+ * @param const CAmount& actual_target This is the value that we want to select. It is the lower
```

Comment:
Perhaps consider calling this `selection_target` or something else that ties it more strongly to the context.

Reply:
Although in that case `nTargetValue` should perhaps also be renamed to something like `selection_target_with_minchange`?

Reply:
Done